src/celeritas/field/CylMapField.hh

```diff
@@ -87,9 +87,9 @@ CELER_FUNCTION auto CylMapField::operator()(Real3 const& pos) const -> Real3
     Real3 value{0, 0, 0};
```

Comment:
You'll probably want  to use `Array<cyl_real_type, 3>` here and convert at the end.